phenotrex/ml/clf/xgbm.py

```diff
@@ -75,6 +75,7 @@ def get_feature_weights(self) -> Dict:
 
     def get_shap(self, records: List[GenotypeRecord],
                  nsamples=None) -> Tuple[csr_matrix, np.ndarray, float]:
```

Comment:
```suggestion
                 n_samples=None) -> Tuple[csr_matrix, np.ndarray, float]:
```
In other places, we use this style. Would improve consistency.